js/components/search.js

```diff
@@ -5,85 +5,165 @@ $( document ).ready(function() {
   //
   // By Sean Herron <sean@herron.io>
   //
-  function GetURLParameter(sParam) {
-    var sPageURL = window.location.search.substring(1);
-    var sURLVariables = sPageURL.split('&');
-    for (var i = 0; i < sURLVariables.length; i++) {
-      var sParameterName = sURLVariables[i].split('=');
-      if (sParameterName[0] == sParam) {
-        return sParameterName[1];
-      }
-    }
```

Comment:
The reason I rewrote this function, even though it worked fine, was because it was harder for me to see what it was actually doing compared to the one I wrote. Granted, some of this is personal style; I am interested in learning how y'all feel about the kind of code I replaced it with.